CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
     - required __init__.py files to support unit test discovery
     - resolve failing unit test cases
     - resolve the virtual machine `__refresh` method
+    - resolve the virtual machine `power_off` method to be consistent with power_on
```

Comment:
Need to remove this line from CHANGELOG.md, as power_off api is not released in the earlier version

Reply:
Thanks Hemant,

Updated.